share/ci/scripts/install_openjph.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+# SPDX-License-Identifier: BSD-3-Clause
+# Copyright Contributors to the OpenColorIO Project.
+
+set -ex
+
+TAG="$1"
+
+if [[ $OSTYPE == "msys" ]]; then
+    SUDO=""
+else
+    SUDO="sudo"
+fi
+
+git clone -b add-export https://github.com/palemieux/OpenJPH.git
```

Comment:
Should this be `https://github.com/aous72/OpenJPH`?

Reply:
Yes. It will be a tagged release of https://github.com/aous72/OpenJPH. I am waiting to ask the maintainer to tag the release until we made sure that no further major changes to the OpenJPH build system is needed to accommodate OpenEXR. Let me know if we are at this point.